### PR TITLE
[Update] add PE link

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-overview.md [hunk 1]
```diff
@@ -44,3 +45,4 @@ Since TiDB is compatible with the MySQL protocol and MySQL syntax, most of the O
 - [Transaction](/develop/dev-guide-transaction-overview.md)
 - [Optimize](/develop/dev-guide-optimize-sql-overview.md)
 - [Example Applications](/develop/dev-guide-sample-application-spring-boot.md)
+- Learn TiDB in-depth and get TiDB certified on [PingCAP Education](https://en.pingcap.com/education/?utm_source=docs-dev-guide)
```
Comment (Oreoxmt) [inline develop/dev-guide-overview.md:48]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@qiancai PTAL

File: develop/dev-guide-overview.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ Before you start working with TiDB, you need to understand some important mechan
 
 - Read the [TiDB Transaction Overview](/transaction-overview.md) to understand how transactions work in TiDB, or check out the [Transaction Notes for Application Developers](/develop/dev-guide-transaction-overview.md) to learn about transaction knowledge required for application development.
 - Understand [the way applications interact with TiDB](#the-way-applications-interact-with-tidb).
+- To learn the core components and concepts of building up the distributed TiDB cluster and TiDB Cloud, refer to the free online course [TiDB Architecture Basics](https://freeclasses.edu.pingcap.com/catalog/info/id:203/?utm_source=docs-dev-guide).
```
Comment (qiancai) [inline develop/dev-guide-overview.md:19]:
```suggestion
- To learn the core components and concepts of building up the distributed database TiDB and TiDB Cloud, refer to the free online course [TiDB Architecture Basics](https://freeclasses.edu.pingcap.com/catalog/info/id:203/?utm_source=docs-dev-guide).
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**